Makefile

```
@@ -211,6 +222,25 @@ catalog-push: catalog-tag-sha ## Push a catalog image.
 	$(CONTAINER_RUNTIME) push $(PUSH_OPTIONS) $(CATALOG_IMG_LATEST)
 	$(CONTAINER_RUNTIME) push $(PUSH_OPTIONS) $(CATALOG_IMG_SHA)
 
+## package-operator package
+
+# The image tag given to the resulting package image
+PACKAGE_IMG_BASE ?= $(IMAGE_BASE)-package
+PACKAGE_IMG ?= $(PACKAGE_IMG_BASE):$(VERSION)
```

should we allow overriding this?
 
```suggestion
PACKAGE_IMG = $(PACKAGE_IMG_BASE):$(VERSION)
```